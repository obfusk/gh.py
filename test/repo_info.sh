#!/bin/bash
"$PY" gh.py repo_info rack/rack 2>&1
