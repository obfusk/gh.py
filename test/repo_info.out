==> https://api.github.com/repos/rack/rack
[
  {
    "name": "rack/rack",
    "desc": "A modular Ruby web server interface.",
    "link": "https://github.com/rack/rack"
  }
]
